### PR TITLE
tests: drivers: build_all: led: Add i2c devices build test

### PR DESCRIPTION
**Pull request by soburi**
Add build tests for following devices.

- issi,is31fl3194
- issi,is31fl3216a
- issi,is31fl3733

### CODE REVIEW

File: tests/drivers/build_all/led/app.overlay [hunk 1]
```diff
@@ -104,6 +104,26 @@
...
+			};
+
+			is31fl3216a@f {
+				compatible = "issi,is31fl3216a";
+				reg = <0xf>;
+			};
+
+			is31fl3733@10 {
+				reg = <0x10>;
+				compatible = "issi,is31fl3733";
```
Comment (simonguinot) [inline tests/drivers/build_all/led/app.overlay:125]: Cosmetic suggestion: you can move `compatible` above `reg`, to align with other nodes

File: tests/drivers/build_all/led/app.overlay [hunk 2]
```diff
@@ -104,6 +104,26 @@
 				compatible = "onnn,ncp5623";
 				reg = <0xd>;
 			};
+
+			is31fl3194@e {
+				compatible = "issi,is31fl3194";
+				reg = <0xe>;
+				led_rgb {
+					label = "led_rgb";
+					color-mapping = <0>, <1>, <2>;
```
Comment (simonguinot) [inline tests/drivers/build_all/led/app.overlay:113]: Eventually, you can use the `LED_COLOR_ID_x` notations.

File: tests/drivers/build_all/led/app.overlay (hunk 1 above)
Comment (soburi) [reply tests/drivers/build_all/led/app.overlay:124]: Thank you for your suggestion.
I applied it.

File: tests/drivers/build_all/led/app.overlay (hunk 2 above)
Comment (soburi) [reply tests/drivers/build_all/led/app.overlay:113]: The overall approach in the `build_all` tests is to compile without semantic consideration,
so I want to leave intentionally make these parameters "nonsense" here as well.

Comment (simonguinot) [reply tests/drivers/build_all/led/app.overlay:113]: OK, but in `tests/drivers/build_all/led_strip/app.overlay` you used the `LED_COLOR_ID_x` notation... I am fine with both approaches but they must be consistent :)

Comment (soburi) [reply tests/drivers/build_all/led/app.overlay:113]: Thank you for your comment.

Yes, I am aware of that...

The code already in led_strip was written like that, so I followed that.
But I think this PR way is the correct way to handle build_all...
There seems to be some difference of opinion among developers on this point...

Comment (simonguinot) [reply tests/drivers/build_all/led/app.overlay:113]: Since I have no opinion it is fine by me. Thanks for the explanation.

**Review by simonguinot — APPROVED** [review]

**Review by danieldegrasse — APPROVED** [review]

**Pull request merged**